high_card.rb

```diff
@@ -3,3 +3,9 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+deck = []
+player = []
+ranks.each { |rank| put decks << ranks}
```

Comment:
This line is giving me an error... specifically there's no `decks` variable (you do have a `deck` variable though... maybe you meant to do that?)